designcompose/src/main/java/com/android/designcompose/DesignFrame.kt

```diff
@@ -89,24 +89,23 @@ internal fun DesignFrame(
     // (row/column/etc) to the replacement component at some point.
     val replacementComponent = customizations.getComponent(name)
     if (replacementComponent != null) {
-        replacementComponent(
-            object : ComponentReplacementContext {
-                override val layoutModifier = layoutInfo.selfModifier
-                override val appearanceModifier = m
-
-                @Composable
-                override fun Content() {
-                    content()
-                }
-
-                override val textStyle: TextStyle? = null
+        val replacementParentLayout =
+            parentLayout?.withReplacementLayoutData(viewStyle.externalLayoutData())
+        DesignParentLayout(replacementParentLayout) {
+            replacementComponent(
+                object : ComponentReplacementContext {
+                    override val layoutModifier = layoutInfo.selfModifier
```

Comment:
Do we need a ticket to look at what's in this Modifier post Rust-layout? (I'm not sure what we have in here anymore).

Reply:
I started looking into this crash more https://github.com/google/automotive-design-compose/issues/864 and I'm in this area of the code so I was going to take a pass at refactoring/removing bits of this that aren't needed anymore.

Reply:
sg, thanks Rob.